trunk/SUAVE/Analyses/Aerodynamics/AVL_Inviscid.py

```diff
@@ -1,4 +1,3 @@
-## @ingroup Analyses-Aerodynamics
```

Comment:
Revert these changes to put back in doc strings.

Reply:
I reverted it. Probably a mistake while I was retyping certain issues. Issue resolved